classes/4.html

```diff
@@ -71,6 +71,122 @@ <h3>State vs Props</h3>
         <!-- Handling Events -->
         <section>
           <h3>Manejo de Eventos</h3>
+
+          <section>
+            <p>
+              Los eventos son acciones que ocurren y provocan algún cambio o
+              notificación en el Sistema.
+            </p>
+            <p>
+              Cuando programamos nuestra página, nosotros decidimos sobre qué
+              eventos queremos reaccionar y cómo.
+            </p>
+          </section>
+
+          <section>
+            <h4>Tipos de Eventos</h4>
+            <p>Existen muchos tipos de eventos distintos, por ejemplo:</p>
+            <ul>
+              <li>Hacer click sobre algún elemento</li>
+              <li>Presionar alguna tecla</li>
+              <li>Pasar el mouse por arriba de algún elemento (hover)</li>
+              <li>Cambiar algún campo de un formulario (change)</li>
+              <li>Apretar el botón de confirmar de un formulario (submit)</li>
+            </ul>
+          </section>
+
+          <section>
+            <h4>Detectar Eventos en React</h4>
+            <p>
+              Una forma de detectar el evento <b>click</b> es usando la
+              propiedad <b>onClick</b> de los elementos
+            </p>
+            <pre><code data-line-numbers data-trim class="language-html">
+				<button onClick={this.manejadorClick}>
```

Comment:
En las clases anteriores creo que no vieron nada sobre `this`. Quizás haya que agregar una pequeña explicación sobre eso.

Reply:
Esto no es lo primero de React, no me quedaba claro en que tema se menciona, si lo tengo que decir en handling events o si se explica antes dado este temario: 

```
		   <ol>
            <li>¿Qué es React?</li>
            <li>Componentes</li>
            <ol>
              <li>JSX</li>
              <li>Ciclo de Vida</li>
              <li>State vs Props</li>
              <li>Manejo de Eventos</li>
              <li>Renderizado Condicional</li>
            </ol>
          </ol>
```

Decis que llegamos a Manejo de Eventos sin haber dicho nada del this?

Reply:
o si esta bien repetirlo igual, ya que aca lo uso, por mas que ademas se mencione algo, volver a decirlo aca?

Reply:
Ah buen punto. Con clases anteriores me refería a las 1 y 2, más que a los slides anteriores. Pero tiene sentido, simplemente sugiero dejar anotado para agregar eso en algun slide anterior